code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -804,9 +804,9 @@
 		draw_color ||= (species_color) || (skin_tone && skintone2hex(skin_tone))
 
 	if(draw_color)
-		limb.color = "[draw_color]"
+		limb.color = "[draw_color][num2hex(owner.dna.species.specific_alpha, 2)]"
```

Comment:
First skyrat comments, and second, what is going on here?

Reply:
This part of the code generates overlay sprites for the limbs in question. `draw_color` is simply an RGB value. However, I read up in the DM ref that `var/color` on `/image` also supports an alpha component. So, I'm essentially appending the carbon's `species.specific_alpha` as a 2 digit hex value onto the RGB color as an alpha component.

Skyrat comments I can do in a second.